report/Dockerfile

```diff
@@ -16,38 +16,45 @@
 # limitations under the License.
 #
 
-FROM ubuntu:16.04
-LABEL maintainer="Maciej Laskowski <skejven@gmail.com>"
+FROM httpd:alpine
+LABEL maintainer="Jan Helak <https://github.com/heljan>"
```

Comment:
Could you please update to `maintainers`?
```suggestion
LABEL maintainers="Jan Helak <https://github.com/heljan>, Maciej Laskowski <skejven@gmail.com>"
```

Reply:
corrected